language-support/scala/codegen-sample-app/src/test/scala/com/digitalasset/codegen/ScalaCodeGenIT.scala

```diff
@@ -62,6 +62,8 @@ class ScalaCodeGenIT
   override implicit lazy val patienceConfig: PatienceConfig =
     PatienceConfig(timeout = Span(20, Seconds), interval = Span(250, Millis))
 
+  // Needs a non-serial execution context because beforeAll() is blocking
+  // Does not have issues with Bazel, as test are not actually asynchronous due to the use of whenReady()
   override implicit def executionContext: ExecutionContext = ExecutionContext.global
```

Comment:
Perhaps we can move this to the `beforeAll` specifically?

Reply:
I _think_ this entire suite should use `WordSpec` and not `AsyncWordSpec`. All tests block using `ScalaFutures.whenReady`, so there is no need for `AsyncWordSpec`.

@leo-da @S11001001

Reply:
@rautenrieth-da, yes `WordSpec` should be used.